server/src/services/semanticHighlight/highlighters/StringHighlighter.ts

```diff
@@ -0,0 +1,25 @@
+import { SemanticTokenTypes } from "vscode-languageserver-protocol";
+import { TokenKind } from "@nomicfoundation/slang/kinds";
+import { NodeType } from "@nomicfoundation/slang/cst";
+import { HighlightVisitor } from "../HighlightVisitor";
+import { SlangNodeWrapper } from "../../../parser/slangHelpers";
+
+const stringKinds = new Set([
```

Comment:
Maybe I'm missing something, but aren't these things highlighted by the syntactic (textmate) default highlighter? Why do we need to go through the LSP to define them?

- `KeywordHighlighter`
- `NumberHighlighter`
- `StringHighlighter`

Reply:
I suggest so, as this means extra work in the LSP, and extra data sent over the wire to the client, without visible user benefits.